dlp/snippets/inspect_content.py

```diff
@@ -1532,6 +1687,32 @@ def inspect_data_to_hybrid_job_trigger(
         default=True,
     )
 
+    parser_table_hotword = subparsers.add_parser(
+        "table_w_custom_hotword",
+        help="Inspect a table and exclude column values when matched "
+             "with custom hot-word.",
+    )
+    parser_table_hotword.add_argument(
+        "--project",
+        help="The Google Cloud project id to use as a parent resource.",
+        default=default_project,
+    )
+    parser_table_hotword.add_argument(
+        "--table_data",
+        help="Dictionary representing a table.",
```

Comment:
please add an example of how a valid argument would be passed in, with a couple of headers and a couple of rows. thanks!

Reply:
Updated Please check